frontends/web/src/locales/en/app.json

```diff
@@ -1758,11 +1758,26 @@
     "status": {
       "complete": "Complete",
       "failed": "Failed",
-      "pending": "Pending"
+      "pending": "Pending",
+      "pending_receive": "Incoming transaction",
+      "pending_send": "Outgoing transaction",
+      "pending_send_to_self": "Pending transaction"
+    },
+    "statusShort": {
+      "complete": "Complete",
+      "failed": "Failed",
+      "pending": "Pending",
```

Comment:
here is some duplication with previous key, reason is we use status on larger screens and statusShort on smaller screens. _receive _send _send_to_self is added by transaction context but the key itself also contains a variable

```js
            {t(`transaction.statusShort.${status}`, {
              context: type
            })}
```

🤔  should I keep or update somehow?